rv-monitor/src/main/javacc/com/runtimeverification/rvmonitor/core/parser/RVParser.jj

```diff
@@ -251,7 +251,7 @@ Specification specification() : {
     (languageParameters = delimitedNoCurly())?
     "{"
         {languageDeclarations = parseUntilLineMatches(Pattern.compile(
-            "^([-a-zA-Z\\s_]*)event([a-zA-Z_\\s0-9]+)\\("));}
+            "^([-a-zA-Z\\s_]*)\bevent\b([a-zA-Z_\\s0-9]+)\\(.*\\)(\\s)*\\{"));}
```

Comment:
What is an example that new code can handle but the old one could not?

Reply:
For example, we have an rvm file
```
unittest () {
    public void event1Handler() throws GotoStmtException, RaiseStmtException {
        ...
    }

    event event1(...) {
        ...
    }
}
```
The old regex will treat the java function `public void event1Handler()` as the event declaration (i.e, it matches the old regex), and then throws parsing error. Now the regex will exclude this case (it requires `event` to be a word with boundary and matches unitl `{`)

Reply:
Thanks for the explanation @yzhang90 !

I wonder if this unittest can be added to rv-monitor's test suite?